### PR TITLE
Remove Secrets

### PR DESCRIPTION
**Pull request by ryuyu**
Address the issues that scanning for credentials on our repository found.

@skofman1 @maartenba

### CODE REVIEW

File: tests/NuGetGallery.Facts/Authentication/AuthenticationServiceFacts.cs [hunk 1]
```diff
@@ -192,7 +192,7 @@ public class TheAuthenticateMethod : TestContainer
                     string.IsNullOrEmpty(ar.UsernameOrEmail)));
             }
 
-            //[Fact]
+            [Fact]
```
Comment (scottbommarito) [inline tests/NuGetGallery.Facts/Authentication/AuthenticationServiceFacts.cs]: Why are we reactivating this test? Why was it deactivated in the first place?

File: src/NuGetGallery/Web.config [hunk 2]
```diff
@@ -52,8 +52,8 @@
     <add key="Gallery.AzureCdnHost" value="" />
     <!-- Set this to the Azure CDN Host you are using for blob storage, if you have configured one -->
     <add key="Gallery.SiteRoot" value="http://nuget.localtest.me/" />
-    <add key="reCAPTCHA::PrivateKey" value="6LcebdwSAAAAAI5dI90LpPMFGKDxdCt6fjwEwMut" />
```
Comment (scottbommarito) [inline src/NuGetGallery/Web.config]: With this change we are losing these keys forever. Do we want to save them?

File: src/NuGetGallery/Web.config [hunk 3]
```diff
@@ -52,8 +52,8 @@
     <add key="Gallery.AzureCdnHost" value="" />
     <!-- Set this to the Azure CDN Host you are using for blob storage, if you have configured one -->
     <add key="Gallery.SiteRoot" value="http://nuget.localtest.me/" />
-    <add key="reCAPTCHA::PrivateKey" value="6LcebdwSAAAAAI5dI90LpPMFGKDxdCt6fjwEwMut" />
-    <add key="reCAPTCHA::PublicKey" value="6LcebdwSAAAAAFI06sH9RRb2VP1HFCjYpg74lKG7" />
+    <add key="reCAPTCHA::PrivateKey" value="" />
+    <add key="reCAPTCHA::PublicKey" value="" />
```
Comment (skofman1) [inline src/NuGetGallery/Web.config]: >PublicKey [](start = 25, length = 9)

we need to place those in KeyVault, and to update octopus to replace them for dev, int. Rotating the key is a good idea as well.

Also, can u please check if you can run Gallery locally, and what is the behavior when those values are missing? We don't want to break customers that use the Gallery as is.

**Comment by ryuyu** [comment]
Reopening against dev.

File: src/NuGetGallery/Web.config (hunk 3 above)
Comment (ryuyu) [inline src/NuGetGallery/Web.config]: Gallery seems to behave okay without them, but i'm not 100% sure where they are used, so i couldn't test all cases. I'm actually okay with leaving these here if they are actually for public use.

**Pull request closed without merging**